src/refinement-list/refinement-list.ts

```diff
@@ -86,7 +89,8 @@ export class NgAisRefinementList extends BaseWidget {
   @Input() public limit: number;
   @Input() public showMore: boolean;
   @Input() public showMoreLimit: number;
-  @Input() public sortBy: string[] | ((item: object) => number);
+  @Input()
+  public sortBy: FacetSortByStringOptions[] | ((item: object) => number);
```

Comment:
The signature of the function should be more like the one defined inside HierarchicalMenu:

https://github.com/algolia/angular-instantsearch/blob/3f79c4b9864f10061a5ff3c9beb707e4496e0c42/src/hierarchical-menu/hierarchical-menu.ts#L53

Reply:
same reply as https://github.com/algolia/angular-instantsearch/pull/557#discussion_r286686683 will do it

Reply:
done here: 80976e3